utils/firestore.js

```diff
@@ -433,6 +413,46 @@ const retrieveParticipantsEligibleForIncentives = async (siteCode, roundType, is
     }
 }
 
+const removeParticipantsDataDestruction = async () => {
+    try {
+        const dataDestructionFieldList = ['104278817', '119449326', '153713899', '173836415', '231676651', '262613359', '268665918', '269050420', '304438543', '359404406', '399159511', '407743866', '412000022', '471168198', '479278368', '524352591', '526455436', '544150384', '558435199', '577794331', '592227431', '613641698', '664453818', '744604255', '747006172', '765336427', '773707518', '826240317', '831041022', '883668444', '996038075', 'token', 'Connect_ID', 'query', 'pin'];
+
+        const currSnapshot = await db
+            .collection('participants')
+            .where('831041022', '==', 353358909)
+            .get();
+
+        for (const doc of currSnapshot.docs) {
+            const batch = db.batch();
```

Comment:
should `batch` be declared outside of the for loop? or else using batching seems pointless, I could be wrong though

Reply:
I tried it and ran into an issue about limited 500 records